scipy/stats/_continuous_distns.py

```diff
@@ -67,6 +67,29 @@ def wrapper(self, *args, **kwds):
     return wrapper
 
 
+def _get_left_bracket(fun, rbrack, lbrack=None):
```

Comment:
We do this sort of thing all the time. It's time to break it out into a function.

Reply:
Yes :)